modules/tensor_mechanics/src/materials/PowerLawCreepStressUpdate.C

```diff
@@ -32,24 +32,24 @@ validParams<PowerLawCreepStressUpdate>()
   params.addParam<Real>("start_time", 0.0, "Start time (if not zero)");
   params.addCoupledVar("temperature", 0.0, "Coupled temperature");
   params.addParam<std::string>(
-      "creep_prepend", "", "String that is prepended to the creep_strain Material Property");
+      "creep_prepend", "creep", "String that is prepended to the creep_strain Material Property");
```

Comment:
Nice, I'm glad this worked!  I'm not crazy about that name, though... Let me think about that a bit.  We should add that to the advanced parameters group so it doesn't show up with the regular params in the documentation.

Reply:
Do you want it separate from creep_prepend to leave that untouched?

Reply:
Maybe we should just have a parameter called `effective_strain_name` in the `RadialReturnStressUpdate` class that has to be defined, and we have the derived classes define that behind the scenes, but still allow that to be user-setttable.  I vaguely remember trying something like that when we originally did it and it didn't work for some reason.

Reply:
it didn't work using `getParam<std::string>`, but it did work with `parameter.get<std::string>`. I'm not positive the difference between the two, or which one to use when. I changed all others to getParam<std::string> in this class to be consistent though because that is what I usually see.

Reply:
Actually, if `effective_strain_name` was set in the `RadialReturnStressUpdate` wouldn't this would require the class to be declared outside of the constructor? (not sure that is even allowed?). `effective_strain_name` could be used in the derived classes to set the constructor, but all that would really do is a be a separate parameter from `creep_prepend`. Seems like this way works just fine?

Reply:
If you mean you aren't crazy about the `creep_prepend` name, I agree with you. I was just using what was already baked in.

Reply:
Yeah, that's right.  I'm not really sure what motivated that, but I don't like how it's specific to this model. We could expand the use of `base_name` into these models, which would solve these issues in a way that is more consistent with everything else in tensor_mechanics.

Reply:
I think that makes sense. Where would that be defined? StressUpdateBase?

Reply:
I just submitted PR #11370 that should clean up these naming conflicts.